### PR TITLE
feat(theme): improve the design

### PR DESCRIPTION
**Pull request by hypeJunction**
skin1
skin3
skin4
skin5
skin6
sni2

### CODE REVIEW

File: views/default/elements/components.css.php [hunk 1]
```diff
@@ -257,17 +300,19 @@
...
-	border-bottom-color: #FFF;
-	border-width: 8px;
-	margin-left: -8px;
+	border-bottom-color: #ffffff;
+	border-width: 0.5rem;
+	margin-left: -0.5rem;
 }
+
 .elgg-comments .elgg-list:before,
 .elgg-river-comments:before {
```
Comment (hypeJunction) [inline views/default/elements/components.css.php]: Use glyph instead

**Comment by hypeJunction** [comment]
I am done here

**Comment by jdalsem** [comment]
LGTM... if you fix the docs that this can be merged

**Comment by jdalsem** [comment]
fixes #10201

**Comment by jdalsem** [comment]
also fixes #10271

**Comment by jdalsem** [comment]
done with this?

**Comment by hypeJunction** [comment]
Yes, I am sure there will be issues along the way, but we can fix them as they show up

**Comment by jdalsem** [comment]
lets see what happens :)

**Pull request merged**